libminifi/test/unit/GeneralUtilsTest.cpp

```diff
@@ -27,55 +27,55 @@
 namespace org::apache::nifi::minifi::test {
 
 // intdiv_ceil
-static_assert(0 == utils::intdiv_ceil(0, 1));
-static_assert(0 == utils::intdiv_ceil(0, 2));
-static_assert(1 == utils::intdiv_ceil(1, 2));
-static_assert(1 == utils::intdiv_ceil(1, 3));
-static_assert(1 == utils::intdiv_ceil(3, 3));
-static_assert(2 == utils::intdiv_ceil(4, 3));
-static_assert(2 == utils::intdiv_ceil(5, 3));
-static_assert(0 == utils::intdiv_ceil(-1, 3));
-static_assert(-1 == utils::intdiv_ceil(-3, 3));
-static_assert(-1 == utils::intdiv_ceil(-4, 3));
-static_assert(2 == utils::intdiv_ceil(-4, -3));
-static_assert(2 == utils::intdiv_ceil(-5, -3));
-static_assert(0 == utils::intdiv_ceil(1, -3));
-static_assert(-1 == utils::intdiv_ceil(5, -3));
-static_assert(3 == utils::intdiv_ceil(6, 2));
-static_assert(-3 == utils::intdiv_ceil(-6, 2));
-static_assert(-3 == utils::intdiv_ceil(6, -2));
-static_assert(3 == utils::intdiv_ceil(-6, -2));
-static_assert(0 == utils::intdiv_ceil(0, -10));
+static_assert(0 == minifi::utils::intdiv_ceil(0, 1));
+static_assert(0 == minifi::utils::intdiv_ceil(0, 2));
+static_assert(1 == minifi::utils::intdiv_ceil(1, 2));
+static_assert(1 == minifi::utils::intdiv_ceil(1, 3));
+static_assert(1 == minifi::utils::intdiv_ceil(3, 3));
+static_assert(2 == minifi::utils::intdiv_ceil(4, 3));
+static_assert(2 == minifi::utils::intdiv_ceil(5, 3));
+static_assert(0 == minifi::utils::intdiv_ceil(-1, 3));
+static_assert(-1 == minifi::utils::intdiv_ceil(-3, 3));
+static_assert(-1 == minifi::utils::intdiv_ceil(-4, 3));
+static_assert(2 == minifi::utils::intdiv_ceil(-4, -3));
+static_assert(2 == minifi::utils::intdiv_ceil(-5, -3));
+static_assert(0 == minifi::utils::intdiv_ceil(1, -3));
+static_assert(-1 == minifi::utils::intdiv_ceil(5, -3));
+static_assert(3 == minifi::utils::intdiv_ceil(6, 2));
+static_assert(-3 == minifi::utils::intdiv_ceil(-6, 2));
+static_assert(-3 == minifi::utils::intdiv_ceil(6, -2));
+static_assert(3 == minifi::utils::intdiv_ceil(-6, -2));
+static_assert(0 == minifi::utils::intdiv_ceil(0, -10));
 
 template<int N, int D, typename = void>
 struct does_compile : std::false_type {};
 
 template<int N, int D>
 struct does_compile<N, D,
```

Comment:
this is a bit misleading: `intdiv_ceil(123, 0)` does compile, it just doesn't compile as a constant expression (because it throws)
```suggestion
struct compiles_as_a_constant_expression<N, D,
```

Reply:
I think the name I suggested would be better, but obviously this has nothing to do with the current change, so I don't mind if it stays as it is.

Reply:
I think the suggested naming is more explicit, updated in https://github.com/apache/nifi-minifi-cpp/pull/1986/commits/68cd493925f83232bd843496eaecf9b472db3bb4